cmd/cerbosctl/put/policy.go

```diff
@@ -59,12 +59,21 @@ func (pc *PolicyCmd) Run(log *zap.SugaredLogger, put *Cmd, ctx *cmdclient.Contex
 		return fmt.Errorf("failed to add or update the policies: %w", err)
 	}
 
-	log.Infof("Uploaded: %d - Ignored: %d", policies.Size(), len(errs))
+	_, err = fmt.Fprintf(k.Stdout, "Uploaded: %d - Ignored: %d\n", policies.Size(), len(errs))
+	if err != nil {
+		return fmt.Errorf("failed to print: %w", err)
+	}
```

Comment:
Adding newline between uploaded and ignored to make it easier to parse. We can ignore the error from `Fprint` because it would be really exceptional to fail writing to stdout and there's nothing we can do to recover from it.

```suggestion
	_, _ = fmt.Fprintf(k.Stdout, "Uploaded: %d\nIgnored: %d\n", policies.Size(), len(errs))
```